docs/index.md

```diff
@@ -0,0 +1,174 @@
+(index)=
+(guides)=
+(howtos)=
+(tutorials)=
+
+# CrateDB Guides and Tutorials
+
+CrateDB is a distributed and scalable SQL database for storing and analyzing
+massive amounts of data in near real-time, even with complex queries. It is
+PostgreSQL-compatible, and based on Lucene. 
+
+This section of the documentation guides you how to use CrateDB in practice.
```

Comment:
I will swap the order of those two paragraphs.

Reply:
Sounds good

Reply:
Now even shorter:

> About how to use CrateDB and CrateDB Cloud in practice.

/cc @matriv ⛑️